core/modules/breadcrumbs/helpers/index.ts

```diff
@@ -5,6 +5,7 @@ import { formatCategoryLink } from '@vue-storefront/core/modules/url/helpers'
 export function parseCategoryPath (categoryPath) {
   let routesArray = []
   for (let category of categoryPath) {
+    if (category.url_path == undefined) continue;
```

Comment:
build failed https://travis-ci.org/github/DivanteLtd/vue-storefront/jobs/699239622 Maybe `isUndefined(category.url_path) || isNull(category.url_path)` from lodash?

Reply:
Thanks. I choose to not use Lodash but simply fix the code (from `==` into `===`).

Reply:
great :) thanks